app/components/CoachCard.tsx

```diff
@@ -106,39 +111,48 @@ export default function CoachCard(props: CoachProps) {
           <h3 className="text-xl font-bold text-slate-500 transition-colors hover:text-vsp-500">
             {name}
           </h3>
-          <h2 className="text-sm font-extralight text-slate-500">
-            {t("languages")}
-            {flagCodes.map((lang, index) => {
-              return (
-                <ReactCountryFlag
-                  key={index}
-                  className="px-1"
-                  style={{
-                    fontSize: "1.2em",
-                  }}
-                  countryCode={lang}
+
+          <section className="inline-flex items-center justify-center gap-2 px-1">
+            <h2 className="text-sm font-extralight text-slate-500">
+              {t("languages")}
+              {flagCodes.map((lang, index) => {
+                return (
+                  <ReactCountryFlag
+                    key={index}
+                    className="px-1"
+                    style={{
+                      fontSize: "1.2em",
+                    }}
+                    countryCode={lang}
+                  />
+                );
+              })}
+            </h2>
+
+            {mhfaTraining ? (
+              <picture className="inline-flex">
```

Comment:
Can you extract the `span` from the `picture` element and apply the `inline-flex` class to a `div` that wraps `picture` and `span`? Because the `picture` HTML element should only contain `source` and `img` as far as I remember. Oh, and you could als add `items-center` to the `inline-flex` to have the text be centered with the image :) Otherwise looks great! 👍

Reply:
Oh, yeah. Sure. :)